### DISCUSSION TITLE
how can i allow j,k repetition.

### DISCUSSION THREAD

**Discussion by semi710** [m4xshen/hardtime.nvim#73] [Q&A] [answered]
Is there any way i can allow j,k repetition or j,k getting pressed more than 15 times in a row?
as sometimes when it's not deterministic how much to scroll j,k scroll makes it easier.

**Comment by kang8** [comment, marked as answer]
you can set `max_count` to 15, which permits repetition up to 15 times;However, this will also apply the 15 times repetition limit to all [`restricted_keys`](https://github.com/m4xshen/hardtime.nvim/blob/4ba3be553fa0b713c7b817f6d201b07d282accf3/lua/hardtime/config.lua#L47-L60), include `h`, `l`, and others.

```lua
-- Hardtime
require('hardtime').setup {
  max_count = 15,
}
```

Alternatively, you can overwrite the `restricted_keys` configuration to remove the restrictions on `j` and `k`. This change will allow an unlimited number of repetitions for these keys without any restrictions.

```lua
-- Hardtime
require('hardtime').setup {
  restricted_keys = {
    ['j'] = {},
    ['k'] = {},
  },
}
```
